coroutines/build.gradle.kts

```diff
@@ -10,6 +10,14 @@ kotlin {
             }
         }
 
+        val jvmTest by getting {
+            dependencies {
+                implementation(Coroutines.test)
+                implementation(kotlin("test"))
```

Comment:
The two kotlin deps would probably be better stuffed in the multiplatform-common

Reply:
Oh right, I already forgot you did that